cypress/integration/nameDetail.spec.js

```diff
@@ -326,27 +326,32 @@ describe('Name detail view', () => {
       cy.wait(2000)
       // Address
       cy.getByTestId('ETH-record-input')
+        .wait(2000)
         .clear({ force: true })
         .type(ADDRESS, { force: true })
         .wait(500)
         // Content
         .getByTestId('content-record-input')
         .clear({ force: true })
+        .wait(500)
         .type(CONTENT, { force: true })
         .wait(500)
         // // Text
         .getByTestId('notice-record-input')
         .clear({ force: true })
+        .wait(500)
         .type(TEXT, { force: true })
         .wait(500)
         // Other Text
         .getByTestId('com.twitter-record-input')
         .clear({ force: true })
+        .wait(500)
         .type(OTHER_TEXT, { force: true })
         .wait(500)
         // Other Address
         .getByTestId('LTC-record-input', { timeout: 10000 })
         .clear({ force: true })
+        .wait(500)
```

Comment:
do we really need all of these waits?

Reply:
cypress test was failing because new text was appended to the existing record without this.